### PR TITLE
Fetch Product

### PR DESCRIPTION
**Pull request by SaurabhChawda**
Updated Product database and fetch data inside the home page using Context.
Screenshot (283)

### CODE REVIEW

File: src/Contexts/ProductsContext.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import { createContext } from "react";
+import { useContext, useEffect, useState } from "react";
+import axios from "axios";
+const ProductsContext = createContext(null);
```
Comment (ShreyashMahajan) [inline src/Contexts/ProductsContext.js:4]: you can use ProductContext, well the above one is having no problem. Just for better calling

**Review by ShreyashMahajan — APPROVED** [review]
review done, and code is very clean and properly used

**Review by neetukumari4858 — APPROVED** [review]
code structure is good.it is good to merge

File: src/Contexts/ProductsContext.js (hunk 1 above)
Comment (SaurabhChawda) [reply src/Contexts/ProductsContext.js:4]: Okay, I will change.
Thankyou

**Pull request merged**